### ISSUE TITLE
Flag emojis showing country letters instead of flags on windows

### ISSUE THREAD

**Issue by hesham-codes** [Daniel-Ioannou/flutter_country_picker#107]
I am building a windows application that utilizes country pickers. My code is as follows:

```
class _CountryPickerButton extends StatefulWidget {
  const _CountryPickerButton();

  @override
  State<_CountryPickerButton> createState() => _CountryPickerButtonState();
}

class _CountryPickerButtonState extends State<_CountryPickerButton> {
  Country selectedCountry = Country.parse('Syria');

  @override
  Widget build(BuildContext context) {
    return TextFormField(
      maxLength: 10,
      inputFormatters: [FilteringTextInputFormatter.digitsOnly],
      keyboardType: TextInputType.number,
      decoration: InputDecoration(
        prefix: _buildPhoneButton(context),
        labelText: 'phone number',
        border: const OutlineInputBorder(),
      ),
    );
  }

  Widget _buildPhoneButton(BuildContext context) {
    return TextButton(
      child: Text('+${selectedCountry.phoneCode}'),
      onPressed: () => showCountryPicker(
        context: context,
        onSelect: (country) => setState(() => selectedCountry = country),
      ),
    );
  }
}

```

It's working okay on Android emulators:

![phone](https://user-images.githubusercontent.com/113170076/215280044-a519948a-107a-4b7f-8479-c189d3daea3a.png)

On windows, however, it's showing letters instead of flags:

![windows](https://user-images.githubusercontent.com/113170076/215280065-c76b498f-6533-4fa8-8879-a9dd652dfaaf.png)

**Comment by jkydevelopment** [comment]
It happens the same on flutter web, this happened to me after upgrading to the latest Flutter version:
```
Flutter 3.7.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision b06b8b2710 (hace 8 días) • 2023-01-23 16:55:55 -0800
Engine • revision b24591ed32
Tools • Dart 2.19.0 • DevTools 2.20.1
```
<img width="293" alt="image" src="https://user-images.githubusercontent.com/26298322/216001423-ee6010e8-dc08-435d-801d-bd14b4641147.png">

**Comment by lublak** [comment]
@jkydevelopment i tried an old version (3.3.10). And it also not showing up correctly. I think windows has some other issues.
Its not showing the Noto Emojis.

**Comment by lublak** [comment]
```
Flutter 3.3.10 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 135454af32 (3 months ago) • 2022-12-15 07:36:55 -0800
Engine • revision 3316dd8728
Tools • Dart 2.18.6 • DevTools 2.15.
```

![grafik](https://user-images.githubusercontent.com/44057030/224433671-8500e11d-fde0-47d2-90a1-49340919f30b.png)

**Comment by Daniel-Ioannou** [comment]
Released in [v2.0.21](https://pub.dev/packages/country_picker/versions/2.0.21)
